src/test/java/org/folio/helper/PurchaseOrderHelperTest.java

```diff
@@ -131,13 +104,20 @@ void resetMocks() throws Exception {
   }
 
   @Test
-  void testPostOpenCompositeOrder() throws Exception {
-    logger.info("=== Test post open composite order ===");
-    JsonObject order = new JsonObject(getMockData(LISTED_PRINT_SERIAL_PATH));
-    CompositePurchaseOrder compositePo = order.mapTo(CompositePurchaseOrder.class);
-    prepareOrderForPostRequest(compositePo);
-    compositePo.setWorkflowStatus(CompositePurchaseOrder.WorkflowStatus.OPEN);
-    compositePo.setId(UUID.randomUUID().toString());
+  @DisplayName("Test POST open composite order")
+  void testPostOpenCompositeOrder() {
+    // Given
+    JsonObject order;
+    try {
+      order = new JsonObject(getMockData(LISTED_PRINT_SERIAL_PATH));
+    } catch (IOException ex) {
+      fail(ex);
+      return;
+    }
```

Comment:
can use use throw in method declaraion instead of try catch

Reply:
if yes, please change other places as well

Reply:
I had some trouble getting the stack trace without it, but it looks like I actually get it now. I don't remember exactly why I did that. I will add throws to the method declaration instead.